### PR TITLE
[DataFactory] Swagger changes for ADF newly released features:

### PR DESCRIPTION
**Pull request by wuchunchunt19**
1. I can load data from o365 by leveraging the REST endpoint of o365
2. Support copying incremental files based on lastModifiedTime for S3/File/Blob
3. Enable ADLS Gen2 connector as both source & sink
4. Support data migration from MongoDB to CosmosDB (MongoDB API)
5. Make Kusto source/sink better and support Azure Data exploer source/sink
6. Support copy from ADLS Gen1 to ADLS Gen2 with ACL
7. serviceUrl @ S3 linked service
8. I can copy from Dynamics AX
9. AAD OAuth for OData Linked Service
10. I can copy from Oracle Service Cloud and Google Adwords

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json [hunk 1]
```diff
@@ -4296,6 +4516,254 @@
...
+      },
+      "required": [
+        "host",
+        "username",
+        "password"
+      ]
+    },
+    "GoogleAdWordsLinkedService": {
+      "x-ms-discriminator-value": "GoogleAdWords",
+      "description": "Google Awords service linked service.",
```
Comment (nschonni) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json]: Awords 0-> AdWords

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json [hunk 2]
```diff
@@ -4296,6 +4516,254 @@
...
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DynamicsAXLinkedServiceTypeProperties"
+        }
+      },
+      "required": [
+        "typeProperties"
+      ]
+    },
+    "DynamicsAXLinkedServiceTypeProperties": {
+      "description": "Dynamics AXlinked service properties.",
```
Comment (nschonni) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json]: AXlinked -> AX linked

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json (hunk 1 above)
Comment (wuchunchunt19) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json]: Thanks for the reminder. Fixed.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json (hunk 2 above)
Comment (wuchunchunt19) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json]: Thanks for the reminder. Fixed.

**Review by veronicagg — CHANGES_REQUESTED** [review]
Thanks for submitting the PR. Have these changes been reviewed by ARM team? 
Based on the title looks like these updates have been released already, is that correct? 
Please note that these changes are on a stable spec and may cause breaking changes in SDKs (or data loss for existing clients).

**Comment by wuchunchunt19** [comment]
Yes, you are correct. These updates are already released in our product, most of them have been released for several months.
Besides, these changes are all newly add-on entity models which won't bring any breaking changes, and it also doesn't impact any existing API. Do we need ARM team review on this?

**Comment by veronicagg** [comment]
@wuchunchunt19 thanks for your response. Regarding the new entity models, just be aware that old/current SDK generated clients will not have them, so if the service responds with the new model the currently released clients won't know about them. New versions of SDKs released from this version of the spec, will contain the update. Thanks!

**Review by veronicagg — APPROVED** [review]

**Pull request merged**